sdk/lib/crt/math/arm/__rt_sdiv64.s

```diff
@@ -1,8 +1,9 @@
 /*
- * COPYRIGHT:         BSD - See COPYING.ARM in the top level directory
- * PROJECT:           ReactOS CRT library
- * PURPOSE:           Implementation of __rt_sdiv64
- * PROGRAMMER:        Timo Kreuzer (timo.kreuzer@reactos.org)
+ * PROJECT:     ReactOS CRT library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Implementation of __rt_sdiv64
+ * COPYRIGHT:   Copyright 2015 Timo Kreuzer <timo.kreuzer@reactos.org>
+ *              Copyright 2021 Raman Masanin <rmasanin@ibagroup.eu>
```

Comment:
Typo in the name
```suggestion
 *              Copyright 2021 Roman Masanin <rmasanin@ibagroup.eu>
```

Reply:
I prefer to use 36927roma@gmail.com instead of work email

Reply:
Fixed